Sources/Testing/Testing.docc/AddingTags.md

```diff
@@ -0,0 +1,54 @@
+# Adding tags to tests
+
+<!--
+This source file is part of the Swift.org open source project
+
+Copyright (c) 2023 Apple Inc. and the Swift project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+Add tags to tests and customize their appearance.
+
+## Overview
+
+A complex package or project may contain hundreds or thousands of tests and
+suites. Some subset of those tests may share some common facet, such as being
+"critical" or "flaky". The testing library includes a type of trait called
+"tags" that can be added to tests to group and categorize them.
+
+Tags are different from test suites: test suites impose structure on test
+functions at the source level, while tags provide semantic information for a
+test that can be shared with any number of other tests across test suites,
+source files, and even test targets.
+
+## Adding tags
+
+To add a tag to a test, use the ``Trait/tags(_:)-yg0i`` or
+``Trait/tags(_:)-272p`` trait. These traits take sequences of tags as arguments,
+and those tags are then applied to the corresponding test at runtime. If they
+are applied to a test suite, then all tests in that suite inherit those tags.
+
+Tags themselves are instances of ``Tag`` and can be expressed as string literals
+or as named constants declared elsewhere in a test target:
```

Comment:
> …declared elsewhere **in a test target**:

Really, tags can be declared anywhere that the `Testing` module can be used, not necessarily limited to the current test target

Reply:
Only as a side effect of being able to link to the testing library in arbitrary targets. We don't recommend you do that!